GDJS/Runtime/scenestack.ts

```diff
@@ -70,6 +70,44 @@ namespace gdjs {
         } else {
           logger.error('Unrecognized change in scene stack: ' + request);
         }
+
+        const allSyncData: GameSaveState = JSON.parse(
+          localStorage.getItem('save') as string
+        );
+
+        if (currentScene._isLoadingRequested && allSyncData) {
+          currentScene
```

Comment:
That's a lot of logic => Put it in a method with a name that describe what this does.